### ISSUE TITLE
upgrade OIDC/SAML/SPI filter adapter to jakarta.servlet 5.0.0

### ISSUE THREAD

**Issue by sushant17784** [keycloak/keycloak#20036]
### Description

It would be great if we could upgrade all the adapters  servlet filters for OIDC, saml  to run with servlet 5.0 (jakarta.servlet) 

### Motivation

I need to upgrade  my servlet application on new version of tomcat 10 jdk 17 

### Details

Use the migration tool https://github.com/apache/tomcat-jakartaee-migration

**Comment by jellisgwn** [comment]
This will become critical... is there any chance of getting an acknowledgment that this will happen, even if isn't possible to commit to a schedule?

**Comment by kevinleturc** [comment]
Will this issue be necessary after #28774?

**Comment by jellisgwn** [comment]
this issue is already done, and provided by `org.keycloak:keycloak-saml-jakarta-servlet-filter-adapter`.

**Comment by shawkins** [comment]
Closing as out-of-date.
